src/components/calculator/index.js

```diff
@@ -12,56 +11,94 @@ export const packEV = async (set) => {
     let uncommons = 0;
     let rares = 0;
     let mythics = 0;
+    let basicLands = 0;
     for (let card of setData) {
-        if (card.rarity === 'common') {
+        if (card.type_line.toLowerCase().includes('basic') || card.type_line.toLowerCase().includes('gate')) {
+            basicLands++;
+        } else if (card.rarity === 'common') {
             commons++;
         } else if (card.rarity === 'uncommon') {
             uncommons++;
         } else if (card.rarity === 'rare') {
             rares++;
-        } else { mythics++; }
+        } else if (card.rarity === 'mythic') {
+            mythics++;
+        }
     }
-    //removing 5 lands from the common slot
-    commons -= 5
+    commons -= basicLands
     //booster pack - 1 land, 10 commons, 3 uncommons, 1 rare or mythic
-    //P(common) where 10 is num of commons in pack
     let probOfComm = comInPack / commons;
-    //P(uncommon) where 3 is num of uncommons in pack
     let probOfUncomm = uncomInPack / uncommons;
-    //P(rare) where 1 is num of rares in pack and .8649 is chance of pulling rare instead of mythic
     let probOfRare = rareInPack / rares;
-    //P(mythic) where 1 is num of mythics in pack and .8649 is chance of pulling mythic instead of rare
     let probOfMythic = (1 - rareInPack) / mythics;
-    let probOfLand = .2
-
-    //make seperate "valuelist" that excludes cards with price < $2
-    const valueCards = setData.filter(card => parseFloat(card.prices.usd) >= 2)
+    let probOfLand = 1 / basicLands
+    let probOfOther = 0;
```

Comment:
or maybe just a func to caclulate 'setProbabilities' that does the counting and probability calculations

Reply:
returns {probOfLand, probOfComm} etc...

Reply:
going to work on doing this soon when I expand my calculations